source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,162 @@
+</section>
+```
 
-When using the `<nav>` tag, add the ARIA landmark `role="navigation"`.
+### Navigational elements
 
-The detail of using the `<nav>` tag is covered in this [blog post](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
+Use `<nav role="navigation">` to wrap groups of links that aren't already in
+another context (e.g. `<footer>`). Common examples of navigation sections are
+menus, tables of contents, and indexes.
 
-### Emphasis
+As with `<section>`, it is advisable to use `aria-labelledby` to label your `<nav>`.
+This helps to distinguish different `<nav>` blocks in the page.
 
-If you want to give the current location emphasis within a navigational list, use a `<strong>` tag.
+The GOV.UK blog has [more information regarding the use of the `<nav>` tag](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
 
-Example:
+### Aside
+
+`<aside role="complementary">` should be used for content related to the primary content
+of the webpage, but which does not constitute the primary content of the page.
+
+Examples include author information, related links and related content.
+
+### Footer
+
+`<footer role="contentinfo">` should be used for the footer of the site.
+
+## Individual element guidance
+
+### Headings
+
+Headings should be in order - for example `<h1>` then `<h2>`, but not `<h1>` then `<h3>`.
+
+There should only be one `<h1>` element in a page.
+
+### Text emphasis
+
+Italics should be avoided according to the [GOV.UK content style guide](https://www.gov.uk/guidance/style-guide).
+Bold can be used sparingly, but can make large blocks of text difficult to read.
+
+Semantically, words voiced with emphasis can be marked up with `<em>`, whereas
+words conveying a sense or urgency or warning should be marked up with `<strong>`.
+Browsers traditionally render these in italics and bold respectively, so you may
+need to override `em` to not use italics. In theory, screen readers could adopt
+a different tone of voice for this markup, though none currently do.
+
+As a rule, avoid using `<i>` or `<b>`, which are only useful in rare cases. It
+is usually better to use the `font-style` and `font-weight` CSS properties.
+
+### Images
+
+Read the [Design System's Images section](https://design-system.service.gov.uk/styles/images/).
+
+### Buttons vs Links
+
+Links should be used for navigating to another page. Buttons should be used for submitting forms
+or interactions within the page (for example, expanding an element). Empty links (`<a href="#">`)
+should therefore be avoided.
+
+Links should not open new tabs or windows by default, but if users choose to to so (via keyboard
+shortcuts or right-click context menu) then we should respect their choice.
+
+### Visually hidden elements
+
+Sometimes it can be helpful to provide additional content for screen reader users
+that might not be needed for sighted users. For example, the 'Skip to main content'
+link covered earlier.
```

Comment:
The skip link is not the best example as it's mostly for keyboard only users although some screen reader users use it as well. So, this particular example is actually needed for sighted users, it's only hidden until it has focus.
(Not a blocker.)

Reply:
Fair enough. Though hard to think of a concise example otherwise - perhaps something like:

```
Sometimes it can be helpful to provide additional content for screen reader users
that might not be needed for sighted users. For example, contextual descriptions
for links, so that they can be understood in isolation:

`<a href="/fishing">Read more <span class="sr-only">about fishing</span></a>".
```

Reply:
Fair enough. Though hard to think of a concise example otherwise - perhaps something like:

```
Sometimes it can be helpful to provide additional content for screen reader users
that might not be needed for sighted users. For example, contextual descriptions
for links, so that they can be understood in isolation:

`<a href="/fishing">Read more <span class="sr-only">about fishing</span></a>".
```